the-collab-lab/tcl-30-whats-near-me
# [TCL30-58] Export icons from Figma into the project

## Description

As a user 

I would like the iconography of the app to be meaningful and unified

So that I can understand the functionality of the interface.

## Related Issue

[TCL30-58](https://the-collab-lab.atlassian.net/browse/TCL30-58)

## Acceptance Criteria

Export the following SVG Icons from Figma:

- bicycle-icon.svg
- car-icon.svg
- close-icon.svg
- fly-to-me-icon.svg
- list-icon.svg
- map-icon.svg
- search-place-icon.svg
- share-location-icon.svg
- train-icon.svg

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

- Add new SVG icons in our app.

## Testing Steps / QA Criteria

1. Go to folder `src/assets`
2. You should see all SVG icons described previously.
